src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -153,6 +163,81 @@ const ImagesTable = () => {
     );
   }
 
```

Comment:
Can we apply the DRY ("Don't repeat yourself") principle to lines 166 through 240? This is a lot of copy/pasted code. The goal is to not conditionally render anything and to re-use/share as much code as possible between the service and on-prem.